unity-environment/Assets/ML-Agents/Examples/3DBall/3DBall.unity

```diff
@@ -13,7 +13,7 @@ OcclusionCullingSettings:
 --- !u!104 &2
 RenderSettings:
```

Comment:
Was it necessary to push this file ?

Reply:
Yes. We are using a new `.bytes` file for 3DBall.